NBitcoin/Crypto/ECDSASignature.cs

```diff
@@ -139,7 +141,7 @@ public static bool IsValidDER(byte[] bytes)
 			}
 			catch(Exception ex)
 			{
-				Utils.error("Unexpected exception in ECDSASignature.IsValidDER " + ex.Message);
+				//Logs.Utils.LogError(default, ex, "Unexpected exception in ECDSASignature.IsValidDER"); //Potential Side Effect

```

Comment:
if you remove this log call here then the catch is a silent-swallower of errors.

Reply:
yeah, I think this is fine, anyway nobody ever could read those logs.

Reply:
what do you mean with nobody could read these logs?

Reply:
Utils.error was doing nothing. And if it did, this specific error message would not help anyone.

Reply:
Do you want to retain logging here then or not? Reason it was commented out is that this causes the unit tests to fail if left in - a weird behavior. I'll look at resolving but not worth it if silent exception is acceptable?

Reply:
IMO if Utils.error was doing nothing, let's make it do something in this PR instead of removing it altogether

Reply:
or just rethrow

Reply:
@NicolasDorier await your steer?